tests/api.v2/model/__snapshots__/Experiment.test.js.snap

```diff
@@ -2,13 +2,7 @@
 
 exports[`model/Experiment getExperimentData works correctly 1`] = `
 Array [
-  "COALESCE(
-      jsonb_object_agg(pipeline_type, jsonb_build_object('params_hash', params_hash, 'state_machine_arn', state_machine_arn, 'execution_arn', execution_arn))
-      FILTER(
-        WHERE pipeline_type IS NOT NULL
-      ),
-      '{}'::jsonb
-    ) as pipelines",
+  "undefined as pipelines",
```

Comment:
I don't think this snapshot should have changed like this, probably something to do with moving the coalesce into helpers, can you fix this test?